.github/mergify.yml

```diff
@@ -105,6 +105,10 @@ queue_rules:
 
 
         {{ body | get_section("## Proposed Changes", "") }}
+      
+      {% for commit in commits | unique(attribute='email_author') %}
+      Co-Authored-By: {{ commit.author }} <{{ commit.email_author }}>
+      {% endfor %}
```

Comment:
Are these changes relevant?

Reply:
No this is some leftover stuff from `stable` that hasn't been backmerged to `unstable` yet (see https://github.com/sigp/lighthouse/commits/stable/), it should be deleted from this PR.